ptx/sec_optimization.ptx

```diff
@@ -492,7 +523,19 @@
         <caption>Running a power line from the power station to an offshore
           facility with minimal cost in <xref ref="ex_opt3">Example</xref></caption>
         <image xml:id="img_opt3b" width="47%">
-          <description></description>
+          <shortdescription>
+            a power line is ran from the power station to an offshore facility
+          </shortdescription>
+          <description>
+            <p>
+            A powerline is run from a powerstation to a offshore facility 5000ft underwater.
```

Comment:
Add a space between 5000 and ft.

Even better: search the source for the `<quantity>` markup used to handle units. The markup is cumbersome, but is designed so that we can use a global publisher switch to decide how the units will look.

If we write `<quantity><mag>5000</mag><base unit="foot"/></quantity>`, we can choose to render this as `5000 ft`, or `5000 feet`, or `5000'`, etc.